.github/workflows/build_rpms.yaml

```diff
@@ -26,17 +26,17 @@ jobs:
       run: |
         echo "REGISTRY_USERNAME=${{ secrets.DOCKER_REGISTRY_USERNAME }}" >> $GITHUB_ENV
         echo "REGISTRY_PASSWORD=${{ secrets.DOCKER_REGISTRY_PASSWORD }}" >> $GITHUB_ENV
-    - name: Build and push RPMs
+    - name: Prepare options file
       run: |
-        echo $REGISTRY_PASSWORD | docker login docker.io --password-stdin --username $REGISTRY_USERNAME
```

Comment:
Removing this may cause the pull to fail since it could hit the unauthenticated pull limit.